.docker/images/nginx/tests/src/RobotsTxtTest.php

```diff
@@ -22,25 +26,26 @@ public function provideHosts()
   }
 
   /**
-   * Disallow in govcms.
+   * Test that robots.txt returns correct Disallow directive for provided hosts.
    *
-   * @dataProvider provideHosts
+   * @dataProvider providerDisallowedHosts
    */
-  public function testDisallowString($host)
-  {
-    $robots_txt = \curl_get_content("/robots.txt", "-H 'Host: $host'");
-    $this->assertEquals($robots_txt[0], "User-agent: *");
-    $this->assertEquals($robots_txt[1], 'Disallow: /');
+  public function testDisallowedHosts($host) {
+    $robots_txt = \curl_get_content('/robots.txt', "-H 'Host: $host'");
+    $this->assertEquals('User-agent: *', $robots_txt[0]);
```

Comment:
swapped places in assertions as the first argument should be an expected value and the second one should be an actual value.